### DISCUSSION TITLE
Publish/Subscribe

### DISCUSSION THREAD

**Discussion by twenzel** [microsoft/garnet#125] [Q&A] [answered]
Does garnet support the Redis pub/sub feature?
Or can garnet be used as backplane for SignalR scale out? (see https://learn.microsoft.com/en-us/aspnet/core/signalr/redis-backplane?view=aspnetcore-8.0)

**Comment by davidglassborow** [comment, marked as answer]
https://microsoft.github.io/garnet/docs/commands/api-compatibility
